.github/workflows/build.yaml

```
@@ -0,0 +1,57 @@
+name: Build Docker image and push to Dockerhub
+
+on:
+  push:
+    branches:
+      - "main"
```

I'm wondering if we can have this build on *every* branch/commit and only push to Docker on `main`. That way we can be sure this builds, and subsequent changes don't break it, before merging.

```suggestion
on:
  push
```